packages/mazeppa/mazeppa.0.4.0/opam

```diff
@@ -0,0 +1,48 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A modern supercompiler for call-by-value functional languages"
+maintainer: ["Lewis Pearson <hirrolot@gmail.com>"]
+authors: ["Lewis Pearson <hirrolot@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/mazeppa-dev/mazeppa"
+doc: "https://github.com/mazeppa-dev/mazeppa"
+bug-reports: "https://github.com/mazeppa-dev/mazeppa/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "dune" {>= "3.14"}
+  "pprint"
+  "checked_oint" {>= "0.2.1"}
+  "ppx_deriving"
+  "ppx_string_interpolation"
+  "ppx_yojson_conv"
+  "ppx_blob" {>= "0.9.0"}
+  "bisect_ppx"
+  "menhir"
```

Comment:
Looks like we are hitting a lower bounds error due to this unconstrained dependency on menhir:

```
#=== ERROR while compiling mazeppa.0.4.0 ======================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.4.14.2 | pinned(https://github.com/mazeppa-dev/mazeppa/archive/refs/tags/v0.4.0.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/mazeppa.0.4.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p mazeppa -j 39 @install
# exit-code            1
# env-file             ~/.opam/log/mazeppa-7-19e53b.env
# output-file          ~/.opam/log/mazeppa-7-19e53b.out
### output ###
# File "lib/dune", line 22, characters 0-26:
# 22 | (menhir
# 23 |  (modules parser))
# (cd _build/.sandbox/49aaa9762fd354fd4b60213f6d1a699d/default && /home/opam/.opam/4.14/bin/menhir lib/parser.mly --base lib/parser --infer-write-query lib/parser__mock.ml.mock)
# /home/opam/.opam/4.14/bin/menhir: unknown option '--infer-write-query'.
```

It looks like this flag was added in version [20180523](https://gitlab.inria.fr/fpottier/menhir/-/blob/master/CHANGES.md?ref_type=heads#20180523)

Let's try

```suggestion
  "menhir" {>= "20180523"}
```